portfolio/src/component/NoOutlineCard/NoOutlineCard.tsx

```
@@ -0,0 +1,83 @@
+        <img
+          src={image}
+          className="w-full h-full object-cover md:hidden"
+          alt={title}
+        />
+      </div>
+
+
+      <div className="mt-4 space-y-2 flex-1 flex flex-col justify-between">
+        <div className="space-y-2">
+          <h3 className="text-2xl text-rose-300 font-bold">{title}</h3>
+          <p dangerouslySetInnerHTML={{ __html: description }} className="base" />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)